src/components/DirectoryTree.tsx

```diff
@@ -317,6 +317,16 @@ export class DirectoryTree extends React.Component<DirectoryTreeProps, {
     this.lastClickedFile = file;
 
   }
+  timeout = 0;
+  layout = () => {
```

Comment:
can you rename `layout` to something like `onLayout`, `onLayoutDOMEvent`, etc. to avoid confusion with util's `layout` which causes this event?

Reply:
Sure. I just copied this code from `components/editor/Editor.tsx`, should I update this one as well?

Reply:
No preference. Sure, you can add change in Editor.tsx to this PR.